app/src/main/res/values/strings.xml

```diff
@@ -174,6 +174,7 @@
   <string name="prefs_highlight_bookmarks_summary">Highlight bookmarked ayahs while reading</string>
   <string name="prefs_translation_text_title">Translation text size</string>
   <string name="prefs_translations">Translations</string>
+  <string name="prefs_more_translations">More Translations</string>
```

Comment:
I was basing this off the pattern used for `R.string.translations` where it referenced `<string name="translations">@string/prefs_translations</string>`. Otherwise I would've just referenced the string directly. Please let me know if this isn't necessary